cloudera-on-centos/README.md

```diff
@@ -1,14 +1,100 @@
+
+
+
+# Readme
+This template creates a multi-server Cloudera CDH 5.4.x Apache Hadoop deployment on CentOS virtual machines, and configures the CDH installation for either POC or high availability production cluster.
+
 The template also provisions storage accounts, virtual network, availability set, network interfaces, VMs, disks and other infrastructure and runtime resources required by the installation.
 
+The template expects the following parameters:
+
+| Name   | Description | Default Value |
+|:--- |:---|:---|
+| adminUsername  | Administrator user name used when provisioning virtual machines | testuser |
+| adminPassword  | Administrator password used when provisioning virtual machines | Eur32#1e |
+| cmUsername | Cloudera Manager username | cmadmin |
+| cmPassword | Cloudera Manager password | cmpassword |
+| storageAccountPrefix | Unique namespace for the Storage Account where the Virtual Machine's disks will be placed | defaultStorageAccountPrefix |
+| numberOfDataNodes | Number of data nodes to provision in the cluster | 3 |
+| dnsNamePrefix | Unique public dns name where the Virtual Machines will be exposed | defaultDnsNamePrefix |
+| region | Azure data center location where resources will be provisioned |  |
+| storageAccountType | The type of the Storage Account to be created | Standard_LRS |
+| virtualNetworkName | The name of the virtual network provisioned for the deployment | clouderaVnet |
+| subnetName | Subnet name for the virtual network where resources will be provisioned | clouderaSubnet |
+| addressPrefix | The network address space for the virtual network | 10.0.0.0/24 |
+| subnetPrefix | The network address space for the virtual subnet | 10.0.0.0/24 |
+| nodeAddressPrefix | The IP address prefix that will be used for constructing private IP address for each node in the cluster | 10.0.0. |
+| tshirtSize | T-shirt size of the Cloudera cluster (Eval, Prod) | Eval |
+| vmSize | The size of the VMs deployed in the cluster (Defaults to Standard_DS14) | Standard_DS14 |
+
+
 Topology
```

Comment:
This seems more important to be read than the EULA. Can you put this above the EULA?